_events/migrations/env.py

```diff
@@ -1,6 +1,6 @@
 from __future__ import with_statement
```

Comment:
This is also `from ... import ...`
Don't you think it's required to move it down as well?